packages/rich-list-core/test/RichListCoreTest.ts

```diff
@@ -0,0 +1,20 @@
+import { JsonRpcClient } from '@defichain/jellyfish-api-jsonrpc'
+import { RichListItem } from '@defichain/rich-list-api-client'
+import { waitForCondition } from '@defichain/testcontainers'
+import { InMemoryDatabase, StubbedQueueClient } from '../src/lib'
+import { CrawledBlock, RichListCore } from '../src/RichListCore'
+
+export function RichListCoreTest (apiClient: JsonRpcClient): RichListCore {
```

Comment:
maybe this can move to RichListCore static create.. :sweat_smile:

Reply:
Thought about it, but the file is (and going to be) quite huge. So decide to make another pure functional constructor